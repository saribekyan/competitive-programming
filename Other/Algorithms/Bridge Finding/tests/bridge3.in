15 20
9 14
3 6
10 8
7 10
1 9
4 10
6 14
2 3
9 5
5 13
6 7
7 9
1 3
14 12
12 11
9 4
7 13
15 13
13 8
10 3
