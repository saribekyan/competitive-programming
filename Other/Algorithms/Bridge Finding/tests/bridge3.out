2 3
11 12
12 14
13 15
